WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -237,6 +237,12 @@ extension UIImage {
         return UIImage.gridicon(.info, size: CGSize(width: 24, height: 24))
     }
 
+    /// Info Outline Icon
+    ///
+    static var infoOutlineImage: UIImage {
```

Comment:
⚠️ Should we also add this to `IconsTests`?

Reply:
oops nice catch! added in https://github.com/woocommerce/woocommerce-ios/pull/2627/commits/9b8a368e4a4905d5a403293276bce9648e9faf92